include/openssl/whrlpool.h

```diff
@@ -16,18 +16,19 @@
 #  define HEADER_WHRLPOOL_H
 # endif
 
-#include <openssl/opensslconf.h>
+# include <openssl/opensslconf.h>
 
 # ifndef OPENSSL_NO_WHIRLPOOL
-# include <openssl/e_os2.h>
-# include <stddef.h>
-# ifdef __cplusplus
+#  include <openssl/e_os2.h>
+#  include <stddef.h>
+#  ifdef __cplusplus
 extern "C" {
-# endif
+#  endif
+#  if !defined(OPENSSL_NO_DEPRECATED_3_0)
 
-# define WHIRLPOOL_DIGEST_LENGTH (512/8)
-# define WHIRLPOOL_BBLOCK        512
-# define WHIRLPOOL_COUNTER       (256/8)
+#   define WHIRLPOOL_DIGEST_LENGTH (512/8)
```

Comment:
I would not deprecate this one

Reply:
Undeprecated this.

Reply:
What's so precious about the digest lengths?

Reply:
Same argument as https://github.com/openssl/openssl/pull/10789#discussion_r365156133